app/interactors/teams.js

```diff
@@ -0,0 +1,25 @@
+const {
+  getTeams,
+  createTeam: createTeamGithub,
+  addMemberToTeam: addMemberToTeamGithub,
+  addMaintainerToTeam: addMaintainerToTeamGithub,
+  deleteTeam: deleteTeamGithub
+} = require('../services/github/teams');
+
+const createTeam = name => createTeamGithub(name).then(resp => resp.data);
+
+const addMemberToTeam = (teamId, username) =>
+  addMemberToTeamGithub({ teamId, username }).then(resp => resp.data);
+
+const addMaintainerToTeam = (teamId, username) =>
+  addMaintainerToTeamGithub({ teamId, username }).then(resp => resp.data);
+
+const deleteTeam = teamId => deleteTeamGithub({ teamId }).then(resp => resp.data);
```

Comment:
All the `resp.data` seems responsability of services instead of the interactors, because is an limitation of service response (All the consumers need data)